exampleSite/data/fr/about.yml

```diff
@@ -7,9 +7,8 @@ about:
     title: Nous sommes créatif
     content: Ceci est avant tout un test
   - icon: ti-cup
-    title: We're Professional
-    content: Lorem ipsum dolor sit amet, consectetur adipisicing elit. Velit, nihil,
-      libero, perspiciatis eos provident laborum eum dignissimos.
+    title: Nous sommes professionnels
```

Comment:
Please can you avoid to push your own tests to the public official repository ? Thanks.

Reply:
I made a report on his profile . Is there any way to block him or something ? @divinerites

Reply:
@mehedi-sharif https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/blocking-a-user-from-your-organization